### DISCUSSION TITLE
e...................

### DISCUSSION THREAD

**Discussion by ZeyLam** [ORelio/Minecraft-Console-Client#2273] [Q&A]
Hi,everyone friendly.When I was joining the server by this client,it returned result in first picture.But the server didn't have whitelist.Do anything wrong in my config?Help me.(qwq
![awawww](https://user-images.githubusercontent.com/114275069/195973581-b71d688c-ceba-4453-8750-53cad163bb1e.png)
)
![for](https://user-images.githubusercontent.com/114275069/195973808-7a2b4f44-9db5-45b6-895b-7a8d7bed7bea.png)

**Comment by BruceChenQAQ** [comment]
Does the server version match the MCC settings?
Can you use a vanilla client to join the server with the same username at the same time?
